processor/transformer/transformer.go

```diff
@@ -118,6 +118,10 @@ func Init() {
 	pkgLogger = logger.NewLogger().Child("processor").Child("transformer")
 }
 
+func isJobTerminated(status int) bool {
+	return status >= 200 && status < 500
```

Comment:
what about `429`?

Reply:
Do we want to handle `429` from transformer? because right now it doesn't return any `429`

Reply:
why shouldn't we handle it already?